tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -208,15 +208,17 @@ void GenerateIntegrationTestsWindowsIISMatrix(params TargetFramework[] targetFra
 
             void GenerateIntegrationTestsWindowsMsiMatrix(params TargetFramework[] targetFrameworks)
             {
-                var targetPlatforms = new[] { "x86", "x64" };
+                var targetPlatforms = new[] { 
+                    (targetPlaform: "x64", enable32Bit: false),
+                    (targetPlaform: "x64", enable32Bit: true),
+                };
 
                 var matrix = new Dictionary<string, object>();
                 foreach (var framework in targetFrameworks)
                 {
-                    foreach (var targetPlatform in targetPlatforms)
+                    foreach (var (targetPlatform, enable32Bit) in targetPlatforms)
                     {
-                        var enable32bit = targetPlatform == "x86";
-                        matrix.Add($"{targetPlatform}_{framework}", new { framework = framework, targetPlatform = targetPlatform, enable32bit = enable32bit });
+                        matrix.Add($"{targetPlatform}_{(enable32Bit ? "32bit" : "64bit")}_{framework}", new { framework = framework, targetPlatform = targetPlatform, enable32bit = enable32Bit });
```

Comment:
Are these `32bit` and `64bit` new? Should we call them something else to allow for `arm64` in the future? Like `x86` and `x64` (or `amd64`). Otherwise `64bit` is ambiguous.

Reply:
Looking at the build logs It looks like this changes the entry from:

```
{
  "x86_net462": {
    "framework": "net462",
    "targetPlatform": "x86",
    "enable32bit": true
  },
  "x64_net462": {
    "framework": "net462",
    "targetPlatform": "x64",
    "enable32bit": false
  }
}
```

to

```
{
  "x64_64bit_net462": {
    "framework": "net462",
    "targetPlatform": "x64",
    "enable32bit": false
  },
  "x64_32bit_net462": {
    "framework": "net462",
    "targetPlatform": "x64",
    "enable32bit": true
  }
}
```

I think the result looks good, as well as the `x64_64bit_net462` and `x64_32bit_net462` names. WDYT @lucaspimentel?

Reply:
BTW these show up as the individual jobs inside the `msi_integration_tests_windows` stage

Reply:
Ohh, I see. Thanks. That's fine.